packages/thumbprint-react/components/Calendar/index.tsx

```diff
@@ -33,13 +16,16 @@ function throwError(message: string): void {
 }
 
 export function normaliseValue(value: CalendarProps['value']): Date[] {
-    if (value === null) {
+    if (value === null || !value) {
```

Comment:
this can just be `if (!value)` because that will also handle the null case

Reply:
Good point. Updated!